cjdns/Makefile

```diff
@@ -17,13 +17,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cjdns
-PKG_VERSION:=v21.1
+PKG_VERSION:=21.1
 PKG_RELEASE:=5
```

Comment:
We might want to reset it to 1

Reply:
Isn't it more like r6 then?

Reply:
Could be, but still I think that adding r should be done via treewide commit